plugins/scaffolder-backend/src/scaffolder/stages/prepare/azure.ts

```diff
@@ -74,4 +99,11 @@ export class AzurePreparer implements PreparerBase {
 
     return path.resolve(tempDir, templateDirectory);
   }
+
+  private getToken(host: string): string | undefined {
+    return (
+      this.scaffolderToken ||
```

Comment:
should we keep prioritising this one when present? or should we prioritise the other one

honest question :)

Reply:
I think this is fine, I'd rather that we keep to as close existinging functionality right now, so that we don't use the integration config key if it doesn't have write permissions etc.

Reply:
Makes sense!